tests/integration/server/semester/semester.service.test.ts

```diff
@@ -377,6 +383,92 @@ describe('Semester Service', function () {
         }
       });
     });
+    describe('faculty', function () {
+      const [lastSpringSemester] = semesters.slice(-1);
+      const testAcademicYear = lastSpringSemester.academicYear + 1;
+      describe(`is ${ABSENCE_TYPE.NO_LONGER_ACTIVE}`, function () {
+        it('carries over from previous available academic year', async function () {
+          const absentFaculty = await facultyRepository
+            .createQueryBuilder('f')
+            .leftJoinAndSelect('f.absences', 'absence')
+            .getOne();
+
+          await absenceRepository.createQueryBuilder()
+            .update(Absence)
+            .set({ type: ABSENCE_TYPE.NO_LONGER_ACTIVE })
+            .where('facultyId = :facultyId', { facultyId: absentFaculty.id })
+            .execute();
+
+          await semesterService.addAcademicYear(testAcademicYear);
+
+          const newFallSemester = await semesterRepository.findOne({
+            where: {
+              term: TERM.FALL,
+              academicYear: testAcademicYear - 1,
+            },
+          });
+          const newFallAbsence = await absenceRepository.findOne({
+            where: {
+              faculty: absentFaculty.id,
+              semester: newFallSemester.id,
+            },
+          });
+          strictEqual(newFallAbsence.type, ABSENCE_TYPE.NO_LONGER_ACTIVE);
+
+          const newSpringSemester = await semesterRepository.findOne({
+            where: {
+              term: TERM.SPRING,
+              academicYear: testAcademicYear,
+            },
+          });
+          const newSpringAbsence = await absenceRepository.findOne({
+            where: {
+              faculty: absentFaculty.id,
+              semester: newSpringSemester.id,
+            },
+          });
+          strictEqual(newSpringAbsence.type, ABSENCE_TYPE.NO_LONGER_ACTIVE);
+        });
+      });
+      describe(`is anything other than ${ABSENCE_TYPE.NO_LONGER_ACTIVE}`, function () {
+        it(`defaults to ${ABSENCE_TYPE.PRESENT}`, async function () {
+          const presentFacultyMember = await facultyRepository
+            .createQueryBuilder('f')
+            .leftJoinAndSelect('f.absences', 'absence')
+            .where('absence.type = :type', { type: ABSENCE_TYPE.PARENTAL_LEAVE })
+            .getOne();
```

Comment:
I think it may be a good idea to rename this variable to `activeFacultyMember` instead of `presentFacultyMember`, since this is supposed to be a faculty member on parental leave.

Reply:
Good point, that lines up better with the test case overall. I made that change in c73bfdd